File: src/main/java/com/app/sportsinsight/model/Player.java
package com.app.sportsinsight.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

import com.app.sportsinsight.enums.Nationality;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@Entity
public class Player {

	@Id
	@GeneratedValue
	private int id;
	private String name;
	private int age;
	//private Sport sport;
	private Nationality nationality;
	private double marketValue;
	
	
}

File: SportsInsightAngular/sports-insight/src/app/model/player.ts
export class Player{

    private id: number;
	private name: string;
	private age: number;
	private marketValue: number;

    public constructor(id: number, name: string, age: number, marketValue: number){
        this.id = id;
        this.name = name;
        this.age = age;
        this.marketValue = marketValue;
    }

    public getId(): number{
        return this.id;
    }

    public getname(): string{
        return this.name;
    }

    public getAge(): number{
        return this.age;
    }

    public getMarketValue(): number{
        return this.marketValue;
    }
}